commit 2c3d0f5a8626a2e932eb2e5546a8c942aeab16a7 (refs/changes/59/459/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-14T13:15:15-07:00 (3 years, 1 month ago)
    
    CMON-9 add cmon builds to purge-mg-builds process
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/tools/purge-mg-builds b/tools/purge-mg-builds
index 964ec17..0271476 100755
--- a/tools/purge-mg-builds
+++ b/tools/purge-mg-builds
@@ -97,6 +97,8 @@ TTL_DAYS_FROM_NAME='{
     "binder": 365,
     "ca": 365,
     "cloudapi": 365,
+    "cmon": 365,
+    "cmon-agent": 365,
     "cn-agent": 365,
     "cnapi": 365,
     "cns": 365,
